### PR TITLE
fix: explainability ease score and sort feat imp

### PR DESCRIPTION
**Pull request by Kleyt0n**

### CODE REVIEW

File: tests/data/small_test_explainability.csv
Comment (crismunoz) [inline tests/data/small_test_explainability.csv:1]: do we need this file?

File: tests/data/small_test_explainability_importance.csv
Comment (crismunoz) [inline tests/data/small_test_explainability_importance.csv:1]: do we need this file?

File: tests/data/small_test_explainability.csv
Comment (kleyt0n) [reply tests/data/small_test_explainability.csv:1]: No. Its a mistake. I will remove

File: tests/data/small_test_explainability_importance.csv
Comment (kleyt0n) [reply tests/data/small_test_explainability_importance.csv:1]: No. Its a mistake. I will remove

**Review by crismunoz — APPROVED** [review]
nice!

**Pull request merged**